WebApi/Controllers/CommentController.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace WebApi.Controllers
 {
-    [Route("api/[controller]")]
+    [Route("api/[controller]/[action]")]
```

Comment:
This is not acceptable today, it is recommended to find another solution to the problem.